src/Gluon.Client/Gluon.Client.csproj

```diff
@@ -39,21 +39,31 @@
   <ItemGroup>
     <Content Include="paket.template" />
   </ItemGroup>
+  <ItemGroup>
+    <Content Include="yarn.lock" />
+  </ItemGroup>
   <PropertyGroup>
     <VisualStudioVersion Condition="'$(VisualStudioVersion)' == ''">12.0</VisualStudioVersion>
   </PropertyGroup>
   <PropertyGroup>
     <RootNamespace>Gluon.Client</RootNamespace>
   </PropertyGroup>
-  <Import Project="..\..\packages\Microsoft.TypeScript.MSBuild\tools\Microsoft.TypeScript.targets" />
+  <Import Project="$(SolutionDir)\packages\Microsoft.TypeScript.MSBuild\tools\Microsoft.TypeScript.targets" Condition="Exists('$(SolutionDir)\packages\Microsoft.TypeScript.MSBuild\tools\Microsoft.TypeScript.targets')" />
   <Import Project="$(MSBuildBinPath)\Microsoft.CSharp.targets" />
   <Import Project="$(MSBuildExtensionsPath32)\Microsoft\VisualStudio\v$(VisualStudioVersion)\WebApplications\Microsoft.WebApplication.targets" Condition="Exists('$(MSBuildExtensionsPath32)\Microsoft\VisualStudio\v$(VisualStudioVersion)\WebApplications\Microsoft.WebApplication.targets')" />
-  <Target Name="AfterBuild" DependsOnTargets="CompileTypeScriptWithTSConfig">
-    <ItemGroup>
-      <JavaScriptFiles Include="dist/Gluon.js" />
-    </ItemGroup>
-    <JavaScriptCompressorTask SourceFiles="@(JavaScriptFiles)" OutputFile="dist/Gluon.min.js" />
-  </Target>
+  <ItemGroup>
+    <ProjectReference Include="..\Gluon\Gluon.fsproj">
+      <Name>Gluon</Name>
+      <Project>{A49BBFB6-98A6-42CA-A5B3-35C89A7C3800}</Project>
+    </ProjectReference>
+    <ProjectReference Include="..\Gluon.CLI\Gluon.CLI.fsproj">
+      <Name>Gluon.CLI</Name>
+      <Project>{72FDBA6E-CE61-4811-9F18-39FAA585B5EA}</Project>
+    </ProjectReference>
```

Comment:
Included the other projects so that they may be packaged in the `npm` package. The sample project shows how to generate the client from an npm script rather than requiring the web server project know where the source code for the client lives. I think this is a better, general solution given you may want multiple front-ends for different purposes.